.github/workflows/tests.yml

```
@@ -83,6 +87,11 @@ jobs:
               core.setOutput('syntax-only', 'false')
             }
 
+            if (label_names.includes('CI-published-commit-bottles')) {
+              console.log('CI-published-commit-bottles label found. Skipping tests job.')
```

```suggestion
            if (label_names.includes('CI-published-bottle-commits')) {
              console.log('CI-published-bottle-commits label found. Skipping tests job.')
```
Since you took my rename suggestion above.